seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/AbstractSeleniumJSRunner.scala

```diff
@@ -28,6 +28,13 @@ abstract class AbstractSeleniumJSRunner(browserProvider: SeleniumBrowser,
     _console = console
   }
 
+  protected def ignoreKeepAlive: Boolean = {
+    val name = code.name
+    name == "frameworkDetector.js" ||
+    name == "testFrameworkInfo.js" ||
+    name == "testMaster.js"
```

Comment:
That's just as fragile as before. Can't we do something that does not rely on knowing magic names of files coming from `scalajs-test-adapter` + sbt plugin?